src/stats.rs

```diff
@@ -322,11 +361,30 @@ async fn get_stats(context: &Context) -> Result<String> {
         securejoin_sources: get_securejoin_source_stats(context).await?,
         securejoin_uipaths: get_securejoin_uipath_stats(context).await?,
         securejoin_invites: get_securejoin_invite_stats(context).await?,
+        timestamps_when_sending_was_enabled,
+        timestamps_when_sending_was_disabled,
     };
 
     Ok(serde_json::to_string_pretty(&stats)?)
 }
 
+async fn get_timestamps(context: &Context, sql_table: &str) -> Result<Vec<i64>> {
+    let res = context
+        .sql
+        .query_map(
+            &format!("SELECT timestamp FROM {sql_table}"),
```

Comment:
I'd add a reasonable LIMIT, say, 1000. It's unlikely that the user will change the setting manually many times